### DISCUSSION TITLE
Path of new GUI Sensors

### DISCUSSION THREAD

**Discussion by crusherweb** [mampfes/hacs_waste_collection_schedule#2307] [Q&A]
Hello all,

after I've updated to the new 2.0 Version I was able to set all up with the GUI instructions. Great Work.
I've also added a sensor which is pretty good working. 
Now I'd like to add some configurations to this sensor. Can you please provide the Path were I can find the created sensor to edit manually?

Thanks
Chris

**Comment by 5ila5** [comment]
You can add and modify sensors of a source by going to `Setting` -> `Devices & Services` -> `Waste Collection Schedule` -> `Configure` button behind the sensor you like to modify the sensors of -> `Sensors To Modify` Select sensors to modify or to add one.

If you want to change the arguments of the source, you can do so by clicking the 3-dot menu -> reconfigure
